### PR TITLE
CB-8455 added --nohooks option

### PR DESCRIPTION
**Pull request by bso-intel**
implemented --nohooks option described in CB-8455

### CODE REVIEW

**Comment by bso-intel** [comment]
Hi Jesse and Vladimir,

Could you review this together with my pull request in 
https://github.com/apache/cordova-lib/pull/353
for cordova-lib?
Two of them should be pulled together for CB-8455.
Thanks.

File: src/cli.js [hunk 1]
```diff
@@ -79,6 +79,7 @@ function cli(inputArgs) {
         , 'silent' : Boolean
         , 'experimental' : Boolean
         , 'noregistry' : Boolean
+        , 'nohooks': [String, Array]
```
Comment (vladimir-kotikov) [inline src/cli.js]: IMO it 'd be easier to always treat this option as array.

**Comment by vladimir-kotikov** [comment]
@bso-intel, could you please update documentation as well

**Comment by bso-intel** [comment]
Hi Vladimir,

Thank you for reviewing.
I addressed your feedback, and updated the help doc, and created one unit test.

**Comment by vladimir-kotikov** [comment]
Merged. Thanks for update, @bso-intel

**Comment by bso-intel** [comment]
Thank you very much for merging, Vladimir.

**Pull request closed without merging**